CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -
 - Class `Tune` for hyperparameter optimization within existing pipeline ([#1200](https://github.com/tinkoff-ai/etna/pull/1200))
 - Add `etna.distributions` for using it instead of using `optuna.distributions` ([#1292](https://github.com/tinkoff-ai/etna/pull/1292))
+- Add support python 3.11 ([#1301](https://github.com/tinkoff-ai/etna/pull/1301))
```

Comment:
Have you tried to rerun all the tests on python 3.11?

Reply:
Shouldn't we also add CI-job with python 3.11?

Reply:
I'll add